experiment/simulation/js/js/shapes.js

```diff
@@ -0,0 +1,114 @@
+import * as THREE from "https://threejsfundamentals.org/threejs/resources/threejs/r115/build/three.module.js";
+import { createMaterials } from "./materials.js";
+export let createCube = function(x, y, z, shapes, scene, point, shapevertex, dragx, dragy, dragz) {
```

Comment:
Why are we not using const to declare createCube and all other functions in this file?

Reply:
modified to using const